azext_iot/central/params.py

```diff
@@ -64,17 +75,38 @@ def load_central_arguments(self, _):
         context.argument(
             "content",
             options_list=["--content", "-k"],
-            help="Configuration for request. "
-            "Provide path to JSON file or raw stringified JSON. "
-            "[File Path Example: ./path/to/file.json] "
-            "[Stringified JSON Example: {'a': 'b'}] ",
+            help="The device template definition. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example: ./path/to/file.json] [Example of stringified JSON: {<Device Template JSON>}]."
+            " The request body must contain CapabilityModel.",
+        )
+
+    with self.argument_context("iot central device-template create") as context:
+        context.argument(
+            "device_template_id",
+            options_list=["--device-template-id", "--dtid"],
+            help="Unique ID for the Device template.",
+        )
+
+    with self.argument_context("iot central device-template delete") as context:
+        context.argument(
+            "device_template_id",
+            options_list=["--device-template-id", "--dtid"],
+            help="ID of the Device template that you want to delete.",
+        )
+
+    with self.argument_context("iot central device-template show") as context:
+        context.argument(
+            "device_template_id",
+            options_list=["--device-template-id", "--dtid"],
+            help="ID of the Device template that you want to show details",
         )
 
     with self.argument_context("iot central api-token") as context:
         context.argument(
             "token_id",
             options_list=["--token-id", "--tkid"],
-            help="Unique ID for the API token. ",
+            help="The IoT Central ID associated with this token. Specify an alphanumeric ID that you'll then use when modifying"
```

Comment:
nit - it helps to specify what you mean by alphanumeric, especially if it requires lowercase. E.g. [a-z][A-Z][0-9]

Reply:
Fixed.